vendor/bat-native-ads/src/bat/ads/internal/browser/browser_manager.h

```diff
@@ -26,6 +26,8 @@ class BrowserManager final {
   void AddObserver(BrowserManagerObserver* observer);
   void RemoveObserver(BrowserManagerObserver* observer);
 
+  bool WasBrowserUpgraded() { return was_browser_upgraded_; }
```

Comment:
how this method is used?

Reply:
This method is used to decide whether we want to remove notification ads after an upgrade

Reply:
Refactored